doc/src/cylc-user-guide/suiterc.tex

```diff
@@ -13,39 +13,50 @@ \subsection{Top Level Items}
 The only top level configuration items at present are the suite title
 and description.
 
-\subsubsection{title}
+\subsubsection{group}
 
-A single line description of the suite. It is displayed in the db viewer
-window and can be retrieved at run time with the
+A single line group name for a suite. It is displayed in the gscan
+window and can be used as a way of grouping related suites together.
 \lstinline=cylc show= command.
 
 \begin{myitemize}
 \item {\em type:} single line string
 \item {\em default:} (none)
 \end{myitemize}
 
-\subsubsection{description}
 
-A multi-line description of the suite. It can be retrieved by the db viewer
-right-click menu, or at run time with the \lstinline=cylc show= command.
+\subsection{[meta]}
+
+Section containing metadata items for this suite. Several items
+(title, description, URL) are pre-defined and are used by the GUI. Others can be 
+user-defined and passed to suite event handlers to be interpreted according to your 
+needs. For example, the value of an "suite-priority" item could determine how an event 
+handler responds to failure events.
+
+
+\subsubsection[title]{ [meta] \textrightarrow title}
+
+A single line description of the suite. It is displayed in the db viewer
+window and can be retrieved at run time with the
+\lstinline=cylc show= command.
 
 \begin{myitemize}
-\item {\em type:} multi-line string
+\item {\em type:} single line string
 \item {\em default:} (none)
 \end{myitemize}
 
-\subsubsection{group}
+\subsubsection[description]{ [meta] \textrightarrow description}
 
-A single line group name for a suite. It is displayed in the gscan
-window and can be used as a way of grouping related suites together.
-\lstinline=cylc show= command.
+A multi-line description of the suite. It can be retrieved by the db viewer
+right-click menu, or at run time with the \lstinline=cylc show= command.
```

Comment:
(as above) just write "It can be retrieved at run time with the cylc show command".